### DISCUSSION TITLE
Funds not showing in Phantom wallet on solana (please help)

### DISCUSSION THREAD

**Discussion by ghost** [wormhole-foundation/wormhole#515] [Problems / bugs] [answered]
I tried to wormhole bridge my Eth to my phantom wallet and followed all the steps. I clicked on the phantom wallet when selecting wallets and didn't manually enter address or anything. Here is the transaction id 51B32Y9h4sHzZ7g7Zx69UBjQ9Ejo7L1MoG4qTLy1ptUtCiPV589EkZsVDCSVoywhFoLvYmvysN42jHR33BY8dqBU . can anyone help or tell me if I did something wrong?
edit: Upon review of the forum I see that I own the WETH as an nft. I was hoping to bridge it to sol for yield farming, any clue on when this will be patched or if there is a workaround, or if  I can send it back through wormhole?

**Comment by SEJeff** [comment]
@seagijoe  Steve is trying to steal your wallet private key or seed phrase.

The wormhole project will **NEVER** ask for your private key or seed phrase. If you give it out, they will steal your funds. The official wormhole page is https://wormholebridge.com/

**Reply by ghost** [reply]
Thank you Jeff! I figured he was.

**Comment by leoluk** [comment, marked as answer]
It's not an NFT - Phantom has (had?) a display bug where it would believe that the wrapped token was an NFT. You should still be able to us it.
